assignment5.txt

```diff
@@ -0,0 +1,20 @@
+1) To call the function, it would look like this:
+
+    /* Misc stuff above the main function, including functions, function
+    * declarations, header statements (like #define and #include), and stuff
+    */
+    int main(){
+        /* we need 2 ints and 1 float to pass to the function. */
+        int a = 6;
+        float b = 0.5;
+        int c = 2;
+        /* the function returns an integer value */
+        printf("%d\n", fxn(a, b, c));
+    }
+    /* Misc stuff after the main function */
+
+2) Looping is usually better, as it doesn't require the computer to store the previous
```

Comment:
right, complexity is key! and now that we're learning about memory management and data structures, you can further understand this by understanding that each time a function is called, a stack is initialized. So having a recursive function can be very memory ineffective.